documentation/asciidoc/accessories/camera/camera_installing.adoc

```diff
@@ -14,41 +14,45 @@ Depending on the model, the camera may come with a small piece of translucent bl
 
 === Enabling the Camera
 
-==== Using the desktop
-
-Select `Preferences` and `Raspberry Pi Configuration` from the desktop menu: a window will appear. Select the `Interfaces` tab, then click on the `enable camera` option. Click `OK`. You will need to reboot for the changes to take effect.
-
-==== Using the command line
-
-Open the `raspi-config` tool from the terminal:
+Before proceeding, we recommend ensuring that your kernel, GPU firmware and applications are all up to date. You'll need an internet connection for this to work correctly.
```

Comment:
Perhaps this should link to https://www.raspberrypi.org/documentation/computers/os.html#using-apt rather than repeating the apt command-lines? :shrug:

Reply:
Though when it's only 2 lines maybe going to a whole new page is kinda annoying? But I don't mind changing it...

Reply:
Depends on how strongly @aallan feels about the "duplication of content" :wink: